packages/printbox-html/printbox-html.0.6/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+authors: ["Simon Cruanes" "Guillaume Bury"]
+maintainer: "simon.cruanes.2007@m4x.org"
+synopsis: "Printbox unicode handling"
+description: """
+Adds html output handling to the printbox package.
+Printbox allows to print nested boxes, lists, arrays, tables in several formats
+"""
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "dune" { >= "2.0" }
+  "printbox" {= version}
+  "tyxml"
```

Comment:
you removed my `"odoc" {with-doc}` addition, which would make the documentation fail without it (doc dependencies are not transitive, same as the test dependencies)

Reply:
I removed it because it wasn't added upstream and opam-publish doesn't automatically fetch from this PR. I'll add it back.